ideascube/serveradmin/backup.py

```diff
@@ -25,6 +24,7 @@ def make_name(format=None):
 class Backup(object):
 
     FORMAT = settings.BACKUP_FORMAT
+    SUPPORTED_FORMATS_AT_CREATION = ('bztar', 'gztar', 'tar')
```

Comment:
Rereviewing this, I'm thinking we could also drop zip restore support: very very fiew backups have been made (I think only by my or so), I think no body is going to try to restore them (it does a backup before restoring, which backup takes ages); and if we really need to restore a zip file, it's at the end only an unzip and put the directory in the right place, which we can do by hand for once.